src/main/java/frc/robot/ShooterConstants.java

```diff
@@ -0,0 +1,14 @@
+package frc.robot;
+
+public class ShooterConstants {
+    public static final int NEO_MAX_SPEED_RPM = 5676;
+    public static final int SHOOT_LOW_MOTOR_ID = 7;
```

Comment:
the motor ids should be moved to a motor id static constant class. This file shouldn't exist; the constants in the subsystem should be in those files and the speeds and ids should be in the appropriate classes in the constants file